vllm/entrypoints/openai/protocol.py

```diff
@@ -326,6 +326,25 @@ def check_guided_decoding_count(cls, data):
         return data
 
 
+class EmbeddingRequest(BaseModel):
+    model: str
+    input: Union[List[int], List[List[int]], str, List[str]]
+    encoding_format: Optional[str] = Field('float', pattern='^(float|base64)$')
+    dimensions: Optional[int] = None
+    user: Optional[str] = None
+    max_tokens: Optional[int] = Field(default=1,
```

Comment:
Agreed with JZ. If this is not configurable, it shouldn't be exposed to the user.

Reply:
Removed